usecases/linearFunction/linear_model_error.py

```diff
@@ -51,6 +52,18 @@ def __call__(self, parameters):
         [f, df] = self.linear_model(parameters)
         return np.concatenate((f-self.data_f, df-self.data_df))
 
+    def evaluate(self, parameters):
+        """Evaluate the model error for a specific experiment
+
+        Args:
+            parameters(ModelParameters): a dictionary type of parameter list defined in bayes (a and b)
+
+        Returns:
+            return an OrderDict of the individual model errors (e.g. per sensor)
+        """
+        [f, df] = self.linear_model(parameters)
+        return OrderedDict([('f', f-self.data_f), ('df', df-self.data_df)])
```

Comment:
I never really knew how to properly initialize a OrderedDict (and didn't bother to look it up so far). Learned something, thanks!